src/pybind/mgr/dashboard/frontend/src/app/shared/models/osd-settings.ts

```diff
@@ -0,0 +1,4 @@
+export class OsdSettings {
+  osdNearFullRatio: number;
+  osdFullRatio: number;
```

Comment:
Trick: if you keep the back-end side names (nearfull_ratio, etc.) you don't need to manually assign each field from the back-end to the front-end structure (you get all mappings for free!). That's automatically done by the Angular httpclient if you specify the class for the observable it returns.

Reply:
Done